### ISSUE TITLE
Add a Qt implementation

### ISSUE THREAD

**Issue by ColinEberhardt** [tastejs/PropertyCross#59]
http://www.qt-project.org/

**Comment by roschern** [comment]
Hi, looks like not much has happened to this. 
Is there still any work ongoing to add Qt to PropertyCross? I think it would be very interesting.
@bobrippling @jeffwar are you still looking at this? 

New website for all Qt related activities is www.qt.io by the way :)

**Comment by Deadolus** [comment]
We are working on this, there is currently a ALPHA-version, optimized for Android: 
https://github.com/bbvch/PropertyCross/tree/qtPort
Feedback is welcome - plan is to port/optimize this in the near future for the other platforms - help and suggestions are welcome.

**Comment by marco-piccolino** [comment]
@Deadolus when will it be published?

**Comment by Deadolus** [comment]
@marco-piccolino Very soon, the app is finished by now and runs accross all platforms. 
The only problem we have left is getting some screenshots for Windows (emulator doesn't run on my pc and a colleague didn't find any time to start up his emulator, more or less). 
Comment again if I you're  interested in more details. 
In summary one could say that Qt-code runs smoothly on all platforms, but looks like Android everywhere. We weren't interested in customizing the Qt-Widgets to get more native look, rather we expected Qt to handle this.

#318 # Android Screenshot:
![screenshot_2015-08-14-15-55-03](https://cloud.githubusercontent.com/assets/4662005/9926162/6e5e757a-5d13-11e5-8c9f-65a2a0dc1bd1.png)

## Ios Screenshot
![ios simulator screen shot 14 aug 2015 13 46 33](https://cloud.githubusercontent.com/assets/4662005/9926165/749c3418-5d13-11e5-93ea-b2f67c776a56.png)

**Comment by Deadolus** [comment]
Having prodded my colleague again we decided to release the Code as is, without pictures so far - as we have had the App running on Windows phone.
Pull request: https://github.com/tastejs/PropertyCross/pull/319

**Comment by marco-piccolino** [comment]
Awesome, that'll definitely be a good starting point.
Thanks for the effort, hopefully it will show Qt's convenience to a larger
audience.
Il 17/set/2015 01:56 PM, "Deadolus" <notifications@github.com> ha scritto:

**Comment by roschern** [comment]
@Deadolus : great work guys!

**Comment by robbbz** [comment]
Should this be closed now, the pr was merged?

**Comment by marco-piccolino** [comment]
I guess it should be added to http://propertycross.com/ first

**Comment by ColinEberhardt** [comment]
Good point, thanks @marco-piccolino - I've added an issue to the website project so that it doesn't get forgotten.
